vendor/github.com/intel/govmm/qemu/qmp.go

```diff
@@ -727,6 +727,47 @@ func (q *QMP) ExecuteBlockdevDel(ctx context.Context, blockdevID string) error {
 	return q.executeCommand(ctx, "x-blockdev-del", args, nil)
 }
 
+func (q *QMP) ExecuteNetdevAdd(ctx context.Context, netdevID, ifname, downscript, script string) error {
```

Comment:
This part should firstly upstream to github.com/intel/govmm, and re-vendor back to kata-containers after it's merged.
And it's better to put vendor commit in seperate commit.

Reply:
Yes. If the QMP part is correct and tests are passed, I will make a PR to govmm, then re-vendor and rebase this PR.